functions/auth.js

```diff
@@ -3,7 +3,7 @@ const admin = require("firebase-admin");
 const serviceAccount = require("./private/cl-dev-pk.json");
 admin.initializeApp({
   credential: admin.credential.cert(serviceAccount),
-  databaseURL: process.env.VITE_APP_DATABASE_URL,
+  databaseURL: import.meta.env.VITE_APP_DATABASE_URL
```

Comment:
@mohdhaider07 is this working? I don't think that it should. Since the Firebase Functions run on a serverless instance and not in vite.

Just show that the tests are working if possible

Reply:
It is working on my pc. Should I undo ?

Reply:
![image](https://user-images.githubusercontent.com/86488954/221654981-c42819be-99d2-4c93-9817-cb3b62b28609.png)